pkg/chartverifier/reportbuilder_test.go

```diff
@@ -43,3 +48,53 @@ func TestSha(t *testing.T) {
 	}
 
 }
+
+func TestFilePackageDigest(t *testing.T) {
+
+	charts := []string{"checks/chart-0.1.0-v3.with-csi.tgz",
+		"checks/chart-0.1.0-v3.valid.tgz",
+		"checks/chart-0.1.0-v2.invalid.tgz",
+		"checks/chart-0.1.0-v3.without-readme.tgz",
+		"checks/chart-0.1.0-v3.with-crd.tgz"}
+
+	for _, chart := range charts {
+		cmd := exec.Command("sha256sum", chart)
+		var out bytes.Buffer
+		cmd.Stdout = &out
+		err := cmd.Run()
+		if err != nil {
+			fmt.Println("error running command:")
```

Comment:
We should probably report the err in the case

Reply:
added some assert.NoError checks in the tests.